src/test/uint256_tests.cpp

```diff
@@ -115,6 +117,8 @@ BOOST_AUTO_TEST_CASE( basics ) // constructors, equality, inequality
     BOOST_CHECK(uint160S("0x"+MaxS.ToString()) == MaxS);
     BOOST_CHECK(uint160S(R1S.ToString()) == R1S);
     BOOST_CHECK(uint160S("   0x"+R1S.ToString()+"   ") == R1S);
+    BOOST_CHECK(uint160S("   0x"+R1S.ToString()+"-trash;%^&   ") == R1S);
```

Comment:
I did, of course, I'm more interested here in your opinion.

Reply:
Since we're modifying the function I'm suggesting more coverage for the cases, based on what other hex parsers were testing for. It's not a difficult or unreasonable request.

Reply:
> But no need to hold up a bugfix based on a test-only minor nit.

agree, this isn't the reason for my NACK.

Reply:
If you think it's redundant, please mark this as resolved.